code/modules/mob/mob_defines.dm

```diff
@@ -39,7 +39,6 @@
 	var/list/ooc_log = list()
 	var/list/dsay_log = list()
 	var/list/interact_log = list()
-
```

Comment:
dont change files unnecessarily, and before committing something to your pr double check, you didn't accidentally leave unnecessary changes, as you can just discard them at that stage instead of going back and fixing them.